archive/type.go

```diff
@@ -6,17 +6,38 @@ import (
 	"github.com/brimsec/zq/zcode"
 	"github.com/brimsec/zq/zdx"
 	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/resolver"
 )
 
 var ErrSyntax = errors.New("syntax/format error encountered parsing zng data")
 
+// TypeIndexer implements the Indexer interface, building an index
+// where each record has one column called "key" whose values
+// are the unique values found in the zbuf.Writer stream
+// from any fields of the incoming records that have the indicated type.
+
```

Comment:
```suggestion
// from any fields of the incoming records that have the indicated type.
```

looks like my edit inserted a newline